emerging_issues.md

```diff
@@ -4,3 +4,20 @@ css_id: emerging_issues
 ---
 
 # Emerging Issues
+
+As the potential cyber threat against vehicles continues to rise, with [one industry survey](https://upstream.auto/upstream-security-global-automotive-cybersecurity-report-2020/) showing attacks increasing by 99% from 2018 to 2019, and by 700% since 2016, Uptane acknowledges a need to stay well-informed on tomorrow's security issues as well as today's. Towards this end, the Uptane community has identified several emerging issues that will significantly influence the framework and its deployment on the months and years ahead. Below we offer a brief summary of those relevant issues, along with a link to the GitHub issue threads for each. We welcome input from all in the Uptane Community in helping us respond to these new challenges.
+
+**Allowing access to ECUs for emergency updates from federal/state/local governments:** As SOTA delivery of updates becomes more sophisticated, government agencies and regulatory bodies, such as the U.S. Department of Transportation or its state or local equivalents, the Department of Homeland Security, or the Federal Emergency Management Agency,  may require automakers to grant them full control over a vehicle in emergency situations. These infrequent but important OTA broadcasts could be due to emergency or disaster routing, scheduled road work, or traffic conditions due to closed exits, failed stoplights or other conditions.  Accommodating this access will require some re-thinking of how Uptane is configured, particularly in how to prioritize delegations, and perhaps to accommodate a dual director set-up. (See Issue #162 at https://github.com/uptane/uptane-standard/issues/162.)
+
+**Security issues related to the use of aftermarket materials**
+Aftermarket companies refurbish and reuse equipment following end-of-life support from OEMs. This means introducing ECUs to a vehicle that the OEM cannot control in any manner. In addition, because aftermarket suppliers may not have access to the original design, they often must reverse engineer the parts to figure out how they work. Such an approach means suppliers are not able to glean all relevant design information about the ECU, particular the institutional knowledge behind the design decisions made by the original development team. In resolving how Uptane can best adapt to this challenge, we must consider questions like the following: How do we deal with aftermarket ECUs that do not have their own Primary? Can aftermarket ECUs leverage an OEMs Director repository? If an aftermarket ECU does have its own Primary, is each capable of controlling a mutually exclusive set of ECUs? Could ownership of the Director be delegated to a third party or owner? (See Issue #200 at https://github.com/uptane/uptane-standard/issues/200.)
```

Comment:
I don't see a change here, but I do see a typo that needs fixing
```suggestion
Aftermarket companies refurbish and reuse equipment following end-of-life support from OEMs. This means introducing ECUs to a vehicle that the OEM cannot control in any manner. In addition, because aftermarket suppliers may not have access to the original design, they often must reverse engineer the parts to figure out how they work. Such an approach means suppliers are not able to glean all relevant design information about the ECU, particularly the institutional knowledge behind the design decisions made by the original development team. In resolving how Uptane can best adapt to this challenge, we must consider questions like the following: How do we deal with aftermarket ECUs that do not have their own Primary? Can aftermarket ECUs leverage an OEMs Director repository? If an aftermarket ECU does have its own Primary, is each capable of controlling a mutually exclusive set of ECUs? Could ownership of the Director be delegated to a third party or owner? (See Issue #200 at https://github.com/uptane/uptane-standard/issues/200.)
```

Reply:
I just added an apostrophe. I guess the github diff couldn't find the change

Reply:
OK